Cktgen/startviewer.sh

```diff
@@ -0,0 +1,142 @@
+#!/bin/bash
```

Comment:
Implementing this script in Python rather than bash could improve readability.

Reply:
Could but that is how it has been done. (We don't really need startviewer.sh or consume_results.sh, I just added them because I was testing just that little piece of code.)
Let's add a issue to remove the shell script. Really we want to run everything locally without containers. That is the major part of the change that we should do.